examples/common/python/config/config.py

```diff
@@ -104,3 +109,27 @@ def parse_configuration_file(filename, variable_map) :
         text = Template(text).substitute(variable_map)
 
     return toml.loads(text)
+
+# -----------------------------------------------------------------
+# -----------------------------------------------------------------
+
+def read_config_from_toml(input_file, config_name = None, confpaths = [".", TCFHOME + "/" + "config"]):
+    """
+    Function to read toml file and returns the toml content as a list
+    Parameters:
+        - input_file is any toml file which need to be read
+        - config_name is particular configuration to pull
+        - confpaths is the directory structure in which the toml file exists
+    """
+    conf_files = [input_file]
+    config = parse_configuration_files(conf_files, confpaths)
+    if config_name is None:
+        return config
+    else :
+        result = config.get(config_name)
+        if result is None:
+            logger.error("%s is missing in toml file %s", config_name, input_file )
+            return None
```

Comment:
We can `return result` as its already `None` in this case.  we can have a common `return result` for if/else case
```
if result is None
    logger.error("%s is missing in toml file %s", config_name, input_file )
return result
```

Reply:
Thanks Manoj. I had only moved this function not looking into the implementation. Fixed.